logprep/framework/pipeline.py

```diff
@@ -251,10 +233,34 @@ def _iterate(self):
     def _enable_iteration(self):
         self._continue_iterating = True
 
-    def _retrieve_and_process_data(self):
-        event = {}
+    def _process_pipeline(self):
```

Comment:
```suggestion
    def _apply_pipeline(self):
```

Reply:
noooooo.... I think apply is not the right verb... but for sure with process_pipeline I have problems too... 

but not as much as with apply :-D